CONTRIBUTING.md

```diff
@@ -46,6 +46,8 @@ github CI jobs. When executed in the main `oqsprovider` directory,
 
 When installing `act` as a github extension, prefix the commands with `gh `.
 
+For more details on our CI setup, please see [CI.md](CI.md).
+
```

Comment:
As separately mentioned, suggest adding a sample command showing contributors how they can "manually" trigger specific runs (no longer automatically triggered on push)

Reply:
Updated to include an example.